flexget/plugins/list/radarr_list.py

```diff
@@ -0,0 +1,378 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *
+
+import logging
+from collections import MutableSet
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.qualities import Requirements
+from flexget.utils.radarr import RadarrAPIService
+
+# Maps (lowercase) Radarr qualities to flexget
+# quality reqirement strings
+QUALITIES_MAP = {
+    'workprint': 'workprint',
+    'cam': 'cam',
+    'telesync': 'ts',
+    'telecine': 'tc',
+    'dvdscr': 'dvdscr',
+    'sdtv': 'sdtv',
+    'dvd': 'dvdrip', # not completely correct
+    'dvd-r': 'dvdrip', # not completely correct
+    'webdl-480p': 'webdl 480p',
+    'bluray-480p': 'bluray 480p',
+    'bluray-576p': 'bluray 576p',
+    'hdtv-720p': 'hdtv 720p',
+    'webdl-720p': 'webdl 720p',
+    'bluray-720p': 'bluray 720p',
+    'hdtv-1080p': 'hdtv 1080p',
+    'webdl-1080p': 'webdl 1080p',
+    'bluray-1080p': 'bluray 1080p',
+    'remux-1080p': 'remux 1080p',
+    'hdtv-2160p': 'hdtv 2160p',
+    'webdl-2160p': 'webdl 2160p',
+    'bluray-2160p': 'bluray 2160p',
+    'remux-2160p': 'remux 2160p',
+    'br-disk': 'remux', # not completely correct
+    'raw-hd': 'remux', # not completely correct
+
+    # No idea of how to map these:
+    # 'regional': 'UNKNOWN'
+}
+
+log = logging.getLogger('radarr_list')
+
+def radarr_quality_to_flexget_quality_req(radarr_quality):
+    """
+    Translates the provided Radarr quality string to a Flexget Requirement instance.
+    Returns None if translation is unsuccessful
+    """
+
+    # QUALITIES_MAP has its keys in lower case
+    radarr_quality = radarr_quality.lower()
+    if not radarr_quality in QUALITIES_MAP:
+        log.warning(
+            'Did not find a suitible translation for Radarr quality \'%s\'',
+            radarr_quality)
+        return None
+
+    flexget_quality_req_string = QUALITIES_MAP[radarr_quality]
+
+    try:
+        return Requirements(flexget_quality_req_string)
+    except ValueError:
+        log.error(
+            'Failed to convert %s into a valid quality requirement',
+            flexget_quality_req_string)
+        return None
+def get_flexget_qualities(profile, cutoff_only=False):
+    quality_requirements = []
+    if cutoff_only:
+        name = profile['cutoff']['name']
+        quality_req = radarr_quality_to_flexget_quality_req(name)
+        if not quality_req is None:
```

Comment:
If you wanna look for the opposite of `None` you could just do `if quality_req`. That's the more pythonic approach to it